### PR TITLE
A few enhancements

### PR DESCRIPTION
**Pull request by Kris-LIBIS**
Thank you for your nice gem. I added two small enhancements that I use regularly in my own code:

- added the Moments#ago method which is simply a shotcut for Moments#difference(Time.now, ...).
- added the Moments::Difference#humanized method that returns human readable string representation such as '3 days, 5 hours and 16 seconds'

### CODE REVIEW

**Review by excpt — APPROVED** [review]
Thank you very much! That's a great addition.

File: lib/moments/difference.rb [hunk 1]
```diff
@@ -92,6 +92,16 @@ def in_years
...
 
+    def humanized
+      diff_parts = to_hash.each_with_object([]) do |(unit, quantity), parts|
+        parts << "#{quantity} #{quantity == 1 ? unit.to_s[0..-2] : unit}" if quantity.positive?
+      end
+      last_part = diff_parts.pop
+      return last_part if diff_parts.empty?
+
+      [diff_parts.join(', '), last_part].join(' and ')
+    end
```
Comment (AlexWayfer) [inline lib/moments/difference.rb:103]: I see no point in this method without I18n support.

File: lib/moments.rb [hunk 2]
```diff
@@ -8,4 +8,8 @@ module Moments
   def self.difference(from, to, mode = :normal)
     Moments::Difference.new from, to, mode
   end
+
+  def self.ago(from, mode = :normal)
+    Moments::Difference.new from, Time.now, mode
+  end
```
Comment (AlexWayfer) [inline lib/moments.rb:14]: It can conflict (not account) with Rails timezones.

Comment (Kris-LIBIS) [reply lib/moments.rb:14]: IMHO if you are using Rails there are plenty of other options to perform Time calculations. The beauty of this gem is its simplicity and independence of any other gem.

The ago method is just a shortcut and I do not see what new issues this causes. If you do not want to calculate against Time.now, feel free to use Difference.new at any time. But if you add a test case that proves your point, I will gladly look into it.

Comment (AlexWayfer) [reply lib/moments.rb:14]: Yeah. For now, it's just a note from me to think and be aware.

Different, non-Rails, projects can use something for timezones too. Especially with databases…

File: lib/moments/difference.rb (hunk 1 above)
Comment (Kris-LIBIS) [reply lib/moments/difference.rb:103]: I agree, it is a quick hack and this was not an attempt to mimic datetime.difference_in_words, but rather a method to provide backend processes and traces a more readable version of the time lapses without the hassle of setting up translations. I can check for the presence of I18n and use the transalation if present, but introducing a dependency on I18n does not feel right either.

The author started this gem as a small and independent time calculation tool. I conceived the humanize methods in that spirit and for end-user facing time info, I will probaly rather combine it with larger and more elaborate ActiveView or equivalent libraries to get the job done. But if it helps anyone to have optional I18n support, why not?

On that topic, I have not used I18n a lot. Do you have any suggestions on how to use I18n in combining the parts with equivalents of ',' and 'and'?

**Pull request merged**